.github/workflows/omm-ci.yaml

```diff
@@ -73,18 +72,10 @@ jobs:
           --health-retries 5
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
-        with:
-          python-version: "3.11"
-      - name: Install dependencies
+      - name: Build docker image
+        run: docker build -t local .
+      - name: Test with pytest on docker image
         run: |
-          python -m pip install --upgrade pip
-          python3 -m pip install -e .[all]
-      - name: Create tables
-        run: flask --app src/OpenMediaMatch/app.py create_tables
-        env:
-          OMM_CONFIG: ../../.devcontainer/omm_config.py
-      - name: Run tests
-        run: py.test src/OpenMediaMatch
-        env:
-          OMM_CONFIG: ../../.devcontainer/omm_config.py
+          docker run --network="host" local sh -c "python -m pip install --upgrade pip ;
```

Comment:
Is --network the fix that allows it to contact the above docker?

If so, is there a Postgres being stood up by docker itself as well?